.knip.jsonc

```
@@ -6,7 +6,8 @@
     // used for `#is-development` conditional import
     "packages/**/{true,false}.ts",
     // file acts as a type test
-    "packages/xstate-svelte/test/interpreterAsReadable.svelte"
+    "packages/xstate-svelte/test/interpreterAsReadable.svelte",
+    "scripts/{typecheck-examples,update-example-*}.js"
```

Maybe this or similar just in case we have other scripts?
```suggestion
    "scripts/{*}.js"
```

---

I was thinking about that but if it had been in the config to start it wouldn't have detected the unused `dev.js` script so might be better to be more specific? Not sure